stacks/epinio/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Does this have to be "deploy", install/uninstall is a better pair.

Reply:
https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md#adding-your-application